pkgs/development/python-modules/marimo/default.nix

```diff
@@ -19,32 +23,33 @@
   uvicorn,
   websockets,
   pyyaml,
-  pytestCheckHook,
+
+  # tests
+  versionCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "marimo";
-  version = "0.9.1";
+  version = "0.9.14";
   pyproject = true;
 
-  disabled = pythonOlder "3.8";
```

Comment:
Any reason for his removal? Marimo does not officially suppr 3.8 anymore

Reply:
I have been removing those lately as they are more annoying than useful.
People often forget to update them.
Also, we technically only support python 3.11 and 3.12. So, as long as the package is supported for those versions, we don't really need to bother.